networkit/cpp/auxiliary/Log.cpp

```diff
@@ -41,6 +43,8 @@ std::string getLogLevel() {
 		return "ERROR";
 	} else if (current == LogLevel::fatal) {
 		return "FATAL";
+	} else if (current == LogLevel::quiet) {
```

Comment:
It seems that this if-block should be a switch-case

Reply:
Makes sense.